src/main/java/websocket/models/Project.java

```diff
@@ -0,0 +1,62 @@
+package websocket.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import java.util.HashMap;
+
+public class Project {
```

Comment:
One of the fields used in the Project Settings dialog is the project's creation date, and it's currently not supported by this Project model. We could either add it here, or if it's not important we can remove it from the dialog window.